gapis/api/vulkan/extensions/khr_surface.api

```diff
@@ -237,6 +237,7 @@ enum SurfaceType {
   SURFACE_TYPE_WAYLAND = 4
   SURFACE_TYPE_XLIB    = 5
   SURFACE_TYPE_MIR     = 6
+  SURFACE_TYPE_GGP     = 8
```

Comment:
Why not 7 here? (logical next number is 7, not 8?)

Reply:
7 is used internally, and this helps us avoid unnecessary incompatibilities.
I have added 7 as "reserved".